common.blocks/page/page.js

```diff
@@ -33,15 +35,25 @@ provide(BEMDOM.decl(this.name, {
                     this.unbindFromWin('scroll', this._disableScroll);
                     this.bindToWin('scroll', this._onScroll);
                 }, this);
+
+                Modal.on(this.domElem, {modName: 'visible', modVal: '*'}, function(e, data) {
+                    this.setMod('hidden', data.modVal);
+                }, this);
             }
         }
     },
+    viewOrientation: function() {
+        if (document.activeElement.tagName === 'INPUT' || document.activeElement.tagName === 'TEXTAREA') return this;
```

Comment:
use `dom.getFocused()` instead of `document.activeElement` (`dom` is a block in bem-core library, so don't forget to add it to deps)